### ISSUE TITLE
Gtk-Message: Failed to load module

### ISSUE THREAD

**Issue by Magicloud** [snapcrafters/gimp#12]
Ubuntu Bionic. When starting gimp from cli. I saw windows and so. Seemed working. But error messages in output:
```
Gtk-Message: Failed to load module "gail"
Gtk-Message: Failed to load module "atk-bridge"
Gtk-Message: Failed to load module "canberra-gtk-module"
```

**Comment by diddledan** [comment]
I don't know what they mean, but as you say the program works fine. Every GNOME-based snap application seems to emit the same messages.

**Comment by m93a** [comment]
My Gimp doesn't even start. It fails with `Failed to load module "atk-bridge"`. I have both `libatk-adaptor` and `libatk-adaptor:i368` installed 😕

**Comment by damiancs** [comment]
I have the exact same problem. What can I do?

**Comment by Hathoute** [comment]
Tried `sudo snap refresh core --beta`, didn't work.
I also get the same messages.
The problem is that I cannot access to Python-Fu.

**Comment by Hathoute** [comment]
I managed to fix the issue.
If you have installed GIMP from Snap, run `sudo snap remove gimp` to uninstall it.
Then install the latest version by running
`sudo add-apt-repository ppa:otto-kesselgulasch/gimp
sudo apt-get update
sudo apt-get install gimp`
PS: if you have ubuntu 18.04, you don't need `sudo apt-get update`

**Comment by ningacoding** [comment]
Problem persists

**Comment by m93a** [comment]
@ningacoding Please provide more info. Have you installed GIMP with `apt` instead of `snap`?

**Comment by ningacoding** [comment]
You right, sorry if i didn't add details, it looks like it's a general problem, not related to Gimp, it also happens to 

Postman 6.5.3~7.0.4 with

> Ubuntu LTS18.04.2
> snap    2.38 pre1
> snapd   2.38 pre1
> series  16
> kernel  4.15.0-46-generic

Throwing exactly the same message:

> Gtk-Message: Failed to load module "gail"
> Gtk-Message: Failed to load module "atk-bridge"
> Gtk-Message: Failed to load module "canberra-gtk-module"

### THE PROBLEM HERE IS SNAP WITH UBUNTU

**Comment by m93a** [comment]
Yes, that's what Hathoute found, his fix also worked for me. Disable `snap` and install Gimp with `apt` and it will be cool 👌

**Comment by diddledan** [comment]
These messages are not indicative of an error. If the snap isn't working for you then please file a new issue with as much information on your setup and how it is failing. Please don't use the snap's issue tracker to promote other packaging systems or their packages, because this isn't the place for that.
